transformations/azerty_qwerty_chars_swap/README.md

```diff
@@ -0,0 +1,26 @@
+# Multilingual Keyboard Change: AZERTY / QWERTY
+This transformation acts like a perturbation by replacing some characters regarding a possible change of keyboard. Given a character `c_qwerty` located at a certain position in a QWERTY keyboard, it is replaced with the character `c_azerty` located at the same position of an AZERTY keyboard.
```

Comment:
Hi @notricenotsweet, please add your name, email and affiliation.

Reply:
Hello Kaustubh, cf my mail